packages/hardhat-plugin/src/index.ts

```diff
@@ -44,23 +31,12 @@ extendConfig((config, userConfig) => {
       config.paths.root,
       userPathsConfig.ignition ?? IGNITION_DIR
     ),
-    deployments: path.resolve(
```

Comment:
I am not sure we should allow restructuring within the Ignition directory. So for the moment only allow users to set the top level Ignition folder.

Reply:
Makes sense